### ISSUE TITLE
Patch vulnerabilities for grafana-image-renderer

### ISSUE THREAD

**Issue by ziler-orca** [grafana/grafana-image-renderer#478]
The current version of `grafana-image-renderer` has various vulnerabilities that need to be patched. Please update list of all affected package in `Anything else we need to know?` section.

**What happened**:
The `grafana-image-renderer` container image has various vulnerabilities.

**What you expected to happen**:
Please update affected packages listed in `Anything else we need to know?` section.

**How to reproduce it (as minimally and precisely as possible)**:
Run a container scan tool.

**Anything else we need to know?**:

List of all the affected packages

```
extend-3.0.0
handlebars-4.0.6
cached-path-relative-1.0.1
jsonpointer-4.0.1
json-schema-0.2.3
shell-quote-1.6.1
minimist-0.0.8;minimist-1.2.0;minimist-1.2.5
tough-cookie-2.3.2
minizip
lodash-4.17.4
hoek-2.16.3
json5-0.5.1
elliptic-6.4.0
fresh-0.5.0
mime-1.3.4
brace-expansion-1.1.6
debug-0.7.4;debug-2.6.1
sshpk-1.11.0
glob-parent-2.0.0;glob-parent-5.1.1
trim-0.0.1
path-parse-1.0.5
ansi-regex-3.0.0;ansi-regex-4.1.0
qs-6.3.2
hawk-3.1.3
minimatch-3.0.3;minimatch-3.0.4
decode-uri-component-0.2.0
lodash-4.17.20;lodash-4.17.4
shelljs-0.7.7
stringstream-0.0.5
request-2.79.0
ajv-4.11.5
minimist-0.0.8;minimist-1.2.0
nanoid-3.1.12
debug-0.7.4;debug-2.6.1;debug-4.2.0
ms-0.7.2
is-my-json-valid-2.16.0
braces-1.8.5
js-yaml-3.8.2
concat-stream-1.5.0
tunnel-agent-0.4.3
```

**Environment**:
- Grafana Image Renderer version: `3.8.4`
- Grafana version: N/A
- Installed plugin or remote renderer service: N/A
- OS Grafana Image Renderer is installed on: N/A
- User OS & Browser: N/A
- Others:

**Comment by solidspark** [comment]
Our Twistlock scans are showing `node` and `jimp` as vulnerable in the latest 3.9.0 release:
```
Image Vulnerabilities
ID             	        Severity	Package Name                  	Package Version
CVE-2023-30581 	        high    	node                          	18.18.2
PRISMA-2023-0021	medium  	jimp                          	0.16.13
```
`node` has a fixed version of 20.8.0 for that CVE, `jimp` has a fixed version of 0.17.5.

**Comment by evictorero** [comment]
Both dependencies were upgraded [here](https://github.com/grafana/grafana-image-renderer/pull/492)
